.github/workflows/views-core.yml

```diff
@@ -0,0 +1,32 @@
+name: Core Views Tests
+on:
+  pull_request:
+    paths:
+      # Direct view changes
+      - 'src/Tribe/Views/V2/Views/**'
+      - 'src/views/v2/**'
+      - 'tests/views_core/**'
+      # Core V2 infrastructure
+      - 'src/Tribe/Views/V2/View.php'
+      - 'src/Tribe/Views/V2/Manager.php'
+      - 'src/Tribe/Views/V2/View_Register.php'
+      - 'src/Tribe/Views/V2/Template.php'
+      - 'src/Tribe/Views/V2/Messages.php'
+      - 'src/Tribe/Views/V2/Url.php'
+      - '.github/workflows/views-core.yml'
```

Comment:
@Here and elsewhere: add the suite configuration file to the watched files: `views_core.suite.dist.yml`